Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
+
 var HOURS_COUNT = 25;
 var HOURS = [];
 for (var i = 0; i < HOURS_COUNT; i++) {
-    var hour = moment().hour(i).second(0).minute(0).format('h:mma');
+    var hour = moment().hour(i).minute(0).second(0).format('h:mma');
```

Comment:
Should avoid defining variables inside `for` loops in java.

Now that we're working inside `ES6`, I think you can do some cool things here:

```js
var HOURS = Array(HOURS_COUNT).keys().map(function (hour) {
  return moment().hour(hour).startOf('hour');
});
```